dora/src/baseline/cannon/codegen.rs

```diff
@@ -587,6 +597,55 @@ where
             .store_mem(bytecode_type.mode(), Mem::Local(offset), REG_RESULT.into());
     }
 
+    fn emit_jump_if(
+        &mut self,
+        bytecode: &BytecodeFunction,
```

Comment:
Not in this PR, but move the `BytecodeFunction` into the struct. No need to pass this reference to so many functions.